charts/enterprise-metrics/values.yaml

```diff
@@ -42,132 +52,175 @@ tokengenJob:
   annotations: {}
   initContainers: []
 
-config:
-  auth_enabled: true
-  auth:
-    type: enterprise
-
-  cluster_name: "{{ .Release.Name }}"
-
-  admin_api:
-    leader_election:
-      enabled: true
-      ring:
-        kvstore:
-          store: "memberlist"
-
-  admin_client:
-    storage:
-      type: s3
-
-  api:
-    response_compression_enabled: true
-
-  compactor:
-    data_dir: "/data"
-    sharding_enabled: true
-    sharding_ring:
-      kvstore:
-        store: memberlist
-
-  license:
-    path: "/license/license.jwt"
-
-  ingester:
-    max_transfer_retries: 0
-    lifecycler:
-      join_after: 0s
-      final_sleep: 0s
-      num_tokens: 512
+enterpriseMetrics:
+  config: |
+    multitenancy_enabled: true
+
+    auth:
+      type: enterprise
+
+    admin_api:
+      leader_election:
+        enabled: true
+        ring:
+          kvstore:
+            store: "memberlist"
+
+    {{- if .Values.mimir.minio.enabled }}
+    admin_client:
+      storage:
+        type: s3
+        s3:
+          endpoint: {{ .Release.Name }}-minio.{{ .Release.Namespace }}.svc:9000
+          bucket_name: enterprise-metrics-admin
+          access_key_id: {{ .Values.mimir.minio.accessKey }}
+          secret_access_key: {{ .Values.mimir.minio.secretKey }}
+          insecure: true
+    {{- end }}
+
+    {{- if .Values.gateway.useDefaultProxyURLs }}
+    gateway:
+      proxy:
+        default:
+          url: http://{{ template "enterprise-metrics.fullname" . }}-admin-api.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}            
+        admin_api:
+          url: http://{{ template "enterprise-metrics.fullname" . }}-admin-api.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        alertmanager:
+          url: http://{{ template "enterprise-metrics.fullname" . }}-alertmanager.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        compactor:
+          url: http://{{ template "enterprise-metrics.fullname" . }}-compactor.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        distributor:
+          url: http://{{ template "enterprise-metrics.fullname" . }}-distributor.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        graphite_querier:
```

Comment:
I wonder if we should remove the graphite proxy rules since we don't yet deploy the resources for these proxies.

Reply:
ack